packages/rum-core/test/performance-monitoring/transaction.spec.js

```diff
@@ -144,5 +144,13 @@ describe('transaction.Transaction', function() {
 
     transaction._start = transaction._start - 10000
     expect(transaction.canReuse()).toBe(false)
+
+    transaction = new Transaction('transaction', 'transaction', {
+      canReuse: true,
+      reuseThreshold: 100
```

Comment:
passing in `startTime`: 0 in here and changing transaction._start = 101 in next line would make this easier to understand

Reply:
This doesn't work, we want to make the start go back not forward.